README.md

```diff
@@ -246,6 +259,8 @@ const setGreeting = async () => {
 };
 ```
 
+This example uses the  `useScaffoldContract ` hook to obtain a contract instance for the  `YourContract ` smart contract. The data property of the returned object contains the contract instance that can be used to call any of the smart contract methods.
```

Comment:
I think the extra space was added by mistake for `useScaffoldContract` and `YourContract`